README.md

```diff
@@ -40,6 +40,7 @@ For Windows, you also need to install following software:
 ## Build
 
 ```
+$ git submodule update --init --recursive # if you just cloned the repository
```

Comment:
Or use the script `scripts/reset-submodule`, which only update the required submodules. Although it may not work on Windows.

Reply:
So maybe not a great idea to propose it by default. ;)